debian/clickhouse-server.init

```diff
@@ -3,7 +3,7 @@
 # Provides:          clickhouse-server
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Required-Start:    $network
+# Required-Start:    $time $network
```

Comment:
Does it introduce any dependency, e.g. will it work in Docker with Ubuntu or CentOS or Alpine?

Reply:
By the way, why do we need network for clickhouse-server and what does it mean?

Also I'm wondering what 

```
# Default-Start:     2 3 4 5
# Default-Stop:      0 1 6
```

means?

Reply:
>Also I'm wondering what
>
># Default-Start:     2 3 4 5
># Default-Stop:      0 1 6
>means?

Those are [run levels](https://linux.die.net/man/7/runlevel) at which the unit will be started, analog in systemd is `multi-user.target` and similar.

Reply:
>By the way, why do we need network for clickhouse-server and what does it mean?

Right now - yes, because it uses Required instead of Should, will address this in the upcomming patch.
But note, that it will likely work anyway, since sysvinit has some default network, it may include just localhost, and I would even say that the system w/o `$network` provided (but not failed) is nonsense.

Reply:
>Does it introduce any dependency, e.g. will it work in barebones Docker with Ubuntu or CentOS or Alpine?

docker does not have any init system, so they are fine.

Reply:
> Those are run levels at which the unit will be started

I doubt we need it at all.
It's ok not to start clickhouse-server on system startup.

Reply:
>It's ok not to start clickhouse-server on system startup.

Can you elaborate?
If there will be no runlevels then the service will not start, and so user will complain that after reboot server had not been started.